web/build.sh

```diff
@@ -103,9 +103,9 @@ DOC_BUILD_EMBED_WEB="${BUILDER_TERM_START}build:embed${BUILDER_TERM_END} and ${B
 DOC_TEST_SYMBOL="actions -${BUILDER_TERM_START}test${BUILDER_TERM_END}, ${BUILDER_TERM_START}upload-symbols${BUILDER_TERM_END}"
 
 builder_describe "Builds Keyman Engine for Web." \
-  "@../common/web/keyman-version build:engine build:embed build:web build:ui" \
-  "@../common/web/input-processor build:engine build:embed build:web" \
-  "@tools/sourcemap-root build:engine build:embed build:web" \
+  "@../common/web/keyman-version build" \
+  "@../common/web/input-processor build" \
+  "@tools/sourcemap-root build" \
```

Comment:
So... turns out the dependency subsystem doesn't seem to trigger for `build:target` pairs if a generic `./build.sh build` command is received instead.  So... I should probably disable the fanciness re: specific targets, even if it'll moderately increase build time where the components aren't needed.

Also, something else I noted:

```
User@User-PC MINGW64 /c/keymanapp/keyman/web (change/web/reorg-source-src-reorg)
$ ./build.sh clean build
## [web] clean:engine starting...
[web] Skipping dependency build common/web/keyman-version for clean:engine
[web] Skipping dependency build common/web/input-processor for clean:engine
[web] Skipping dependency build web/tools/sourcemap-root for clean:engine
## [web] clean:engine completed successfully
## [web] clean:embed starting...
[web] Skipping dependency build common/web/keyman-version for clean:embed
[web] Skipping dependency build common/web/input-processor for clean:embed
[web] Skipping dependency build web/tools/sourcemap-root for clean:embed
```

Since I've been branch-hopping between the 📜 and 🧩 PR sets, I had to do a deep-clean to rebuild things (to ensure that `keyman.js.map` didn't reappear) and found that doing a top-level `clean` wasn't enough - I had to manually rebuild subpackages, since the top-level script couldn't cope.  Figured this observation may be worth note as a builder-script pain point.